tests/testthat/test-rr_journal_issn.R

```diff
@@ -0,0 +1,34 @@
+context("rr_journal_issn")
+
+test_that("rr_journal_issn() works", {
+  skip_on_cran()
```

Comment:
why `skip_on_cran()`?

Reply:
I thought it was necessary when looking at `rredlist` tests because I thought `vcr` would not work on CRAN. It is not the case so I removed all calls to `skip_on_cran()` in commit ccfdd6d